### ISSUE TITLE
Have to reset Sonarr API Key on clean build (Docker)

### ISSUE THREAD

**Issue by Lox1on** [Ombi-app/Ombi#4822]
**Describe the bug**
When creating a new Docker build for Ombi, if I use the Setup Wizard and use Plex Authorisation I have to reset the Sonarr API Key or it will not connect. Setup of Radarr and Lidarr work fine in this scenario.

If you run the setup wizard and just create a local Admin logon the problem does not occur.

Note:
When accessing the Sonarr setup in Ombi, if you have setup with Plex Auth the Port field has example text as per the image below with an example.  If you setup with a local admin only that field will only have a 0 (zero) in it.

https://ibb.co/brjzgSq

**To Reproduce**
1. New Docker install created using v4.33
2. Go through the setup Wizard and use Plex Login / Authentication.
3. When in Ombi go to Settings > TV > Sonarr
4. Input Sonarr IP address, Port and API Key (all running locally on the same NAS)
5. Fails to connect to Sonarr.
6. Resetting the API Key in Sonarr and re-entering it in Ombi will fix.

**Expected behavior**
Should just reconnect to Sonarr using the existing API Key.

**Screenshots**

**Logs (Logs directory where Ombi is located)**
Error Log below from a clean Ombi v4.33 Docker install after trying to setup Sonarr

https://pastebin.com/icntppZ

**Desktop (please complete the following information):**
 - OS: Windows 11 / MS Edge
 - Radarr, Sonarr, Lidarr etc running on an Ubuntu 22.04 VM on QNAP NAS
 - Ombi running in a Docker container within QNAP Container Station.
 - Plex Media Server running directly on the QNAP NAS

**Ombi Version (please complete the following information):**
 - Version 4.33 
- Media Server Plex
- Database Type: SQLite

**Additional context**

**Comment by Lox1on** [comment]
In addition to this, which may be related.

When setting up a new Docker using a local admin only I am unable to setup Lidarr.  The Test Connection and Submit buttons are greyed out.  If I setup using Plex Login I can setup Lidarr without a problem.

**Comment by jobrien2001** [comment]
same, resetting sonarr api didnt work for me.

```
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/profile
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/rootfolder
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/rootfolder
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/profile
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/series
fail: Ombi.Schedule.Jobs.Sonarr.SonarrSync[2006]
      Exception when trying to cache Sonarr
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Ombi.Api.Sonarr.SonarrApi.GetSeries(String apiKey, String baseUrl) in /home/runner/work/Ombi/Ombi/src/Ombi.Api.Sonarr/SonarrApi.cs:line 48
         at Ombi.Schedule.Jobs.Sonarr.SonarrSync.Execute(IJobExecutionContext job) in /home/runner/work/Ombi/Ombi/src/Ombi.Schedule/Jobs/Sonarr/SonarrSync.cs:line 52
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/series
fail: Ombi.Schedule.Jobs.Sonarr.SonarrSync[2006]
      Exception when trying to cache Sonarr
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Ombi.Api.Sonarr.SonarrApi.GetSeries(String apiKey, String baseUrl) in /home/runner/work/Ombi/Ombi/src/Ombi.Api.Sonarr/SonarrApi.cs:line 48
         at Ombi.Schedule.Jobs.Sonarr.SonarrSync.Execute(IJobExecutionContext job) in /home/runner/work/Ombi/Ombi/src/Ombi.Schedule/Jobs/Sonarr/SonarrSync.cs:line 52
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/system/status
fail: Ombi.Controllers.V1.External.TesterController[1000]
      Could not test Sonarr
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Ombi.Controllers.V1.External.TesterController.Sonarr(SonarrSettings settings) in /home/runner/work/Ombi/Ombi/src/Ombi/Controllers/V1/External/TesterController.cs:line 415
fail: Ombi.Api.Api[1000]
      StatusCode: NotFound, Reason: Not Found, RequestUri: http://10.0.3.216:8989/api/system/status
fail: Ombi.Controllers.V1.External.TesterController[1000]
      Could not test Sonarr
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Ombi.Controllers.V1.External.TesterController.Sonarr(SonarrSettings settings) in /home/runner/work/Ombi/Ombi/src/Ombi/Controllers/V1/External/TesterController.cs:line 415
```

**Comment by tidusjar** [comment]
What versions of sonarr and Ombi?

**Comment by Lox1on** [comment]
Ombi: v4.33 (in Docker) but did see this in some previous version.  I stayed on 4.29 for a while until I noticed I sorted the work around.

Sonarr: v3.0.9.1549 running on an Ubuntu VM.

**Comment by tidusjar** [comment]
Wait so it’s working on 4.29?

**Comment by Lox1on** [comment]
> Wait so it’s working on 4.29?

I will double check on my Dockers and get back to you shortly, I know I stayed on 4.29 rather than going up ... until I sorted the manual workaround on 4.33.  Will edit this with an update shortly.

**Comment by sephrat** [comment]
I suspect this is the same root cause as https://github.com/Ombi-app/Ombi/issues/4877 which was fixed in v4.35.15.
Can you upgrade and tell us if that fixed it?

**Comment by Lox1on** [comment]
Thanks

All tested and working OK now.
